branch-config.mk.j2

```diff
@@ -45,7 +45,7 @@ COPR_REPO ?= \@rhinstaller/Anaconda-devel
 
 GIT_BRANCH ?= rhel-{$ distro_release $}
 L10N_DIR ?= rhel-{$ distro_release $}
-BASE_CONTAINER ?= registry-proxy.engineering.redhat.com/rh-osbs/ubi{$ distro_release $}:latest
```

Comment:
This change in the template needs to be first introduced to master.
After you merge the PR with the change on master branch you can re-run the checks, and it will be green here.

Reply:
That's why 'Template check' is failing bellow in the tests 
https://github.com/rhinstaller/anaconda/actions/runs/8830009859/job/24242239926?pr=5608

Reply:
Here is it: https://github.com/rhinstaller/anaconda/pull/5614